content/docs/1.10.0/concepts.md

```diff
@@ -344,7 +344,7 @@ After a DR volume is activated, it becomes a normal Longhorn volume and it canno
 
 ## 3.4. Backupstore Update Intervals, RTO, and RPO
 
-Incremental restoration is usually triggered by the periodic backupstore update. You can set the update interval on the backup target settings page (**Setting > Backup Target**).
+Incremental restoration is usually triggered by the periodic backupstore update. You can set the update interval on the backup target settings page (**Backup and Restore > Backup Targets**).
```

Comment:
Can you help update the `Setting > Backup Target` in file [content/docs/1.10.0/snapshots-and-backups/backup-and-restore/set-backup-target.md](http://github.com/longhorn/website/blob/master/content/docs/1.10.0/snapshots-and-backups/backup-and-restore/set-backup-target.md)?